Example/Examples/CollectionViewExample/HorizontalListFactory.swift

```diff
@@ -16,18 +16,21 @@ struct HorizontalListFactory {
 
     func make() -> [CollectionViewSection] {
         let section = StaticCollectionSection(
-            cellBuilders: cellBuilders()
+            cellBuilders: cellBuilders(),
+            spacingBetweenItems: (5.0, 5.0),
```

Comment:
Can't we use parameter labels here?
```suggestion
            spacingBetweenItems: (horizontal: 5.0, vertical: 5.0),
```

Reply:
sure

Reply:
fixed